src/example-relay/next.config.js

```diff
@@ -13,11 +13,11 @@ module.exports = (withCustomBabelConfigFile(
     '@adeira/graphql-relay',
     '@adeira/js',
     '@adeira/monorepo-utils',
+    '@adeira/murmur-hash',
```

Comment:
@michalsanger Why is the murmur hash needed here?

Reply:
As I understand this, during a build within a monorepo, `@adeira` packages are not transpiled and we have to explicitly list them here. The `murmur-hash` is used in `sx` that's how it gets into build.

Reply:
Ay caramba. So it's a dependency of a dependency. That's what I was afraid of. 🙈 

I'd argue that this package (`next-transpile-modules`) no longer serves the purpose we were using it for. You have to be explicit like this since version 5 but what we really want to do is to transpile the `@adeira/*` scope and that's it (without enumerating every possible and even deeply nested workspace 🤦).

I will talk with the authors and see if we should fork it or what. Thanks!

Reply:
Asked here: https://github.com/martpie/next-transpile-modules/issues/143

Reply:
We should be able to get this programmatically using `yarn workspaces info` and grabbing all dependencies and nested dependencies.

Reply:
did you find an alternate package / solution for this ?

Reply:
Since we are transpiling workspaces from this monorepo, we decided to fetch it programmatically using the `yarn workspaces info` command, you can see the PR (here)[https://github.com/adeira/universe/pull/1462]